composer.json

```diff
@@ -12,7 +12,8 @@
         "doctrine/dbal": "^2.9",
         "fideloper/proxy": "^4.0",
         "laravel/framework": "5.7.*",
-        "laravel/tinker": "^1.0"
+        "laravel/tinker": "^1.0",
```

Comment:
Laravel tinker and Carbon are already part of the core framework, so no need to add them again.